src/axom/inlet/docs/sphinx/quick_start.rst

```diff
@@ -130,3 +130,12 @@ own style.  The link below shows the example output from the ``documentation_gen
    :maxdepth: 1
 
    example1_expected_documentation
+
+Inlet also provides a utility for generating a `JSON schema <https://json-schema.org/>`_ from your input file schema.
+This allows for integration with text editors like Visual Studio Code, which allows you to associate a JSON schema
+with an input file and subsequently provides autocompletion, linting, tooltips, and more.
+
+Using the same  ``documentation_generation.cpp`` example, the automatically generated schema can be used to assist
```

Comment:
Added an issue so I don't forget that we should have a dedicated page for `Writer` classes.

https://github.com/LLNL/axom/issues/500